tests/posts_tests/test_posts_views.py

```diff
@@ -2,43 +2,61 @@
 from pytest_django.asserts import assertTemplateUsed
 from django import urls
 from django.contrib.auth.models import User
-
 from conftest import EMPTY_COMMENT, LONG_COMMENT, VALID_COMMENT
 
 
 @pytest.mark.django_db
 class TestViews:
-    def test_feed_view(self, client):
-        response = client.get('/')
-        assert response.status_code == 200
 
-    def test_about_view(self, client):
-        response = client.get('/about/')
-        assert response.status_code == 200
+    def test_feed_view_loaded(self, client):
+        feed_response = client.get('/')
+        assert feed_response.status_code == 200
+        assert 'posts/feed.html' in feed_response.template_name
+
+    def test_about_view_loaded(self, client):
+        about_response = client.get('/about/')
+        assert about_response.status_code == 200
+        assertTemplateUsed(about_response, 'posts/about.html')
 
-    def test_resume_detail_view(self, client, persist_resume):
+    def test_resume_detail_view_redirection_and_template(self, client, persist_resume):
         resume_detail_path = f'/post/{persist_resume.pk}/'
         resume_detail_response = client.get(resume_detail_path)
         assert resume_detail_response.status_code == 200
+        assert 'posts/post_detail.html' in resume_detail_response.template_name
 
-    def test_poll_detail_view(self, client, persist_poll):
+    def test_poll_detail_view_redirection_and_template(self, client, persist_poll):
         poll_detail_path = f'/post/{persist_poll.pk}/'
         poll_detail_response = client.get(poll_detail_path)
         assert poll_detail_response.status_code == 200
```

Comment:
Redirect usually means that the response status code is 3xx (302 for example).
I see here only a status code of 200, maybe worth to change the test name?

Reply:
Done